rust/template/src/ovsdb_api.rs

```diff
@@ -108,15 +108,23 @@ unsafe fn dump_delta(
     module: *const c_char,
     table: *const c_char,
 ) -> Result<CString, String> {
-    let table_str: &str = unsafe { CStr::from_ptr(table) }
+    let table_str: &str = CStr::from_ptr(table)
```

Comment:
I would have saved a lot of time by avoiding these callbacks that you now removed.
I think this is cleaner too.

Reply:
Sorry, this was less obvious when looking at the diff in the PR.